docs/other/firewall.md

```diff
@@ -0,0 +1,23 @@
+title: Firewall Considerations
+DateReviewed:
+
+Firewall Considerations
+=======================
+
+In order for hosts such as CEs to communicate with the OSG Factory hosts,
+firewall rules should be added to allow the following connections:
```

Comment:
Since the most important traffic is factory -> CE, I'd swap the two. Also, go ahead and generalize this paragraph more since I'd like to add more sections below for OSDF hosts and squids

Reply:
Updated, maybe have another once-over to see if it looks ok to you